### DISCUSSION TITLE
Error manifest in docker pulll

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [azavea/raster-vision#1708] [Q&A] [answered]
Hello, there!
First of all, thanks for this amazing tool!
I am trying to test using docker, but once I run `docker pull quay.io/azavea/raster-vision`, I get the following error:
```
Using default tag: latest
Error response from daemon: manifest for quay.io/azavea/raster-vision:latest not found: manifest unknown: manifest unknown
```

Am I missing something?
Thanks in advance!

**Comment by AdeelH** [comment, marked as answer]
The correct "latest" tag is `pytorch-latest`. This is the build for the current `master` branch. The tag `pytorch-0.20` is the build for the latest `v0.20.*` release.

So you actually want to do:
```sh
docker pull quay.io/azavea/raster-vision:pytorch-latest
```
or
```sh
docker pull quay.io/azavea/raster-vision:pytorch-0.20
```

**Reply by FelipeSBarros** [reply]
Thanks @AdeelH !
Worked fine.
The sugestion from azavea made me confusing.
![image](https://user-images.githubusercontent.com/7208674/218756126-dc23c494-ca42-4550-b475-eb6c9c20c18b.png)
Best regards

**Reply by AdeelH** [reply]
I agree this is confusing. We should really fix the discrepancy. Thanks for the suggestion!

**Reply by FelipeSBarros** [reply]
Don't worry. Thanks again for your time. I will try to set this post as "solved".
